rdt-open-reader/src/main/res/values/strings_in.xml

```diff
@@ -0,0 +1,50 @@
+<resources>
+    <string name="app_name">RDT Open Reader</string>
+
+    <string name="clients_register_title">Klien</string>
+    <string name="hamburger_menu">Menu hamburger</string>
+    <string name="launching_expiration_date_widget">Meluncurkan tanggal masa berlaku widget</string>
+    <string name="phone_state_permissions_required">Izin ke status ponsel diperlukan untuk mengirimkan formulir ini</string>
+    <string name="rdt_permissions_required">Pengambilan gambar RDT membutuhkan izin ke kamera dan penyimpanan</string>
+    <string name="location_permissions">Izin lokasi diperlukan untuk melanjutkan</string>
+    <string name="time_expired">Waktu telah habis.\nBaca hasil RDT.</string>
+    <string name="login_app_name">RDT Open Reader</string>
+    <string name="login_enter_username">Masukkan nama pengguna</string>
+    <string name="login_enter_password">Masukkan kata kunci</string>
+    <string name="login_attributions"> <string name=\"login_attributions\">Sebuah pengembangan dari <![CDATA[<b><font color=#467690>Ona</font></b>]]> yang <![CDATA[<br/>]]>dipersembahkan oleh <![CDATA[<b><font color=#467690>OpenSRP</font></b>]]></string></string>
```

Comment:
@shivenbhatt1 I think the problem here is `<string name="login_attributions"> <string name=\"login_attributions\">` which should just be `<string name="login_attributions">`.

Reply:
The file name and location is also wrong.

Reply:
This entire line should read `<string name="login_attributions">Sebuah pengembangan dari <![CDATA[<b><font color=#467690>Ona</font></b>]]> yang <![CDATA[<br/>]]>dipersembahkan oleh <![CDATA[<b><font color=#467690>OpenSRP</font></b>]]></string>`.